### ISSUE TITLE
Data files for tutorials? 

### ISSUE THREAD

**Issue by gdcutting** [PacktPublishing/Hands-On-Artificial-Intelligence-on-Google-Cloud-Platform#1]
This repo has code files for the book tutorials, but not the data files (.csv). The Ch. 2 tutorial seems to use data from a Kaggle dataset which had some columns eliminated, but the other tutorials use .csv files which I don't see anywhere here or else online. Thanks

**Comment by vidyabhandary** [comment]
I agree. Need the data files. I came up to Chapter 2 to find the data file missing.

**Comment by XABUNO** [comment]
Hello, could you point me to the assumed Ch. 2 modified Kaggle dataset? 
> This repo has code files for the book tutorials, but not the data files (.csv). The Ch. 2 tutorial seems to use data from a Kaggle dataset which had some columns eliminated, but the other tutorials use .csv files which I don't see anywhere here or else online. Thanks

**Comment by Sandy4321** [comment]
good question
